src/wp-includes/script-loader.php

```diff
@@ -1855,59 +1855,10 @@ function wp_print_delayed_inline_script_loader() {
 	$wp_scripts = wp_scripts();
 
 	if ( $wp_scripts->has_delayed_inline_script() ) {
-		$output            = <<<JS
-(function () {
-  var nonce = document.currentScript.nonce;
-
-  /**
-   * Load event handler.
-   *
-   * @param {Event} event Event.
-   */
-  function onScriptLoad(event) {
-    var i, len, newScript, matches, scripts;
-    if (
-      !(
-        event.target instanceof HTMLScriptElement ||
-        event.target.async ||
-        event.target.defer ||
-        event.target.id
-      )
-    ) {
-      return;
-    }
-    matches = event.target.id.match(/^(.+)-js$/);
-    if (!matches) {
-      return;
-    }
-    scripts = document.querySelectorAll(
-      '[type="text/template"][data-wp-executes-after="' + matches[1] + '"]'
-    );
-    for (i = 0, len = scripts.length; i < len; i++) {
-      if (nonce && nonce !== scripts[i].nonce) {
-        console.error(
-          "CSP nonce check failed for after inline script. Execution aborted.",
-          scripts[i]
-        );
-        continue;
-      }
-      newScript = scripts[i].cloneNode(true);
-      newScript.type = "text/javascript";
-      scripts[i].parentNode.replaceChild(newScript, scripts[i]);
-    }
-  }
-  document.addEventListener("load", onScriptLoad, true);
-
-  window.addEventListener(
-    "load",
-    () => {
-      document.removeEventListener("load", onScriptLoad, true);
-    },
-    { once: true }
-  );
-})();
-JS;
-		wp_print_inline_script_tag( $output, array( 'id' => 'wp-executes-after-js' ) );
+		wp_print_inline_script_tag(
+			file_get_contents( ABSPATH . WPINC . '/js/wp-delayed-inline-script-loader' . wp_scripts_get_suffix() . '.js' ),
```

Comment:
This is probably fine since it should only be loaded once. It would be nice to avoid running this function if we already know that there are no delayed inline scripts so that we avoid an unnecessary filesystem call.

Reply:
This should already be the case due to the `has_delayed_inline_script` check, correct?

https://github.com/10up/wordpress-develop/blob/06daaba6c979cc6063ddb68c09a80068d1fbccb6/src/wp-includes/script-loader.php#L1857-L1862

Reply:
Yep, you're right. It didn't show up in the original GH diff and I only saw that `wp_print_delayed_inline_script_loader` was always hooked to the `wp_print_scripts` action. All good 👍🏻